addons/binding/org.openhab.binding.amazonechocontrol/ESH-INF/thing/thing-types.xml

```diff
@@ -387,4 +426,28 @@
 		<description>Ascending alarm up to the configured volume</description>
 		<state readOnly="false" />
 	</channel-type>
+	<channel-type id="nextReminder" advanced="true">
+		<item-type>DateTime</item-type>
+		<label>Next reminder</label>

```

Comment:
Words in labels must all be in upper case (this is the convention used in openHAB): `Next Reminder`. Comment also applies for next 3 labels below.

Reply:
Changed for all labels :)  not only the new one.